src/coreclr/jit/compiler.cpp

```diff
@@ -5048,6 +5048,10 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
         // Optimize block order
         //
         DoPhase(this, PHASE_OPTIMIZE_LAYOUT, &Compiler::optOptimizeLayout);
+
+        // Conditional to Switch conversion
+        //
+        DoPhase(this, PHASE_Find_Specific_Pattern, &Compiler::optFindSpecificPattern);
```

Comment:
I assume it was a temporal name for the phase, right? needs to be renamed to something like optSwitchRecognition + PHASE_SWITCH_RECOGNITION or something like that